CMakeLists.txt

```diff
@@ -111,6 +120,18 @@ add_custom_command(
     VERBATIM
 )
 
+file(MAKE_DIRECTORY ${CMAKE_BINARY_DIR}/video-plane-display-dmabuf)
+add_custom_command(
```

Comment:
Instead of adding yet another `custom_command()` usage to invoke `wayland-scanner` it would be nicer to factor this out into a CMake function. For example Cog includes [in FindWaylandProtocols.cmake](https://github.com/Igalia/cog/blob/master/cmake/FindWaylandProtocols.cmake#L77) a CMake function which creates the needed targets and adds the generated files as part of the sources for a given target (note that it cannot be imported and used directly here, I mention it as something from where to draw inspiration).

In this case I think it's not a big deal to duplicate the `custom_command()`, but at some point it would be great to factor this out—specially if more protocols get added.

Reply:
Is it acceptable to do this as a follow-up PR?